### PR TITLE
Updating the guestbook example to v1beta3

### PR DESCRIPTION
**Pull request by nikhiljindal**
For https://github.com/GoogleCloudPlatform/kubernetes/issues/6584

### CODE REVIEW

**Comment by roberthbailey** [comment]
@briandorsey FYI

**Comment by nikhiljindal** [comment]
Fixing tests

**Comment by nikhiljindal** [comment]
Tests fixed!

**Comment by wojtek-t** [comment]
cc @piosz

File: examples/guestbook/README.md [hunk 1]
```diff
@@ -96,27 +101,36 @@ CONTAINER ID        IMAGE                                  COMMAND
...
 
-The services in a Kubernetes cluster are discoverable inside other containers via environment variables. 
+The services in a Kubernetes cluster are discoverable inside other containers via environment variables.
 
 Services find the containers to load balance based on pod labels.
 
-The pod that you created in Step One has the label `name=redis-master`. The selector field of the service determines *which pods will receive the traffic* sent to the service, and the port and containerPort (targetPort in v1beta3 and higher APIs) information defines what port the service proxy will run at. 
+The pod that you created in Step One has the label `name=redis-master`. The selector field of the service determines *which pods will receive the traffic* sent to the service, and the port and containerPort (targetPort in v1beta3 and higher APIs) information defines what port the service proxy will run at.
```
Comment (bgrant0607) [inline examples/guestbook/README.md]: Please make v1beta3 the primary text and older APIs in paretheses.

Comment (bgrant0607) [inline examples/guestbook/README.md]: Or remove mention of the old fields entirely, since we're removing the old example, anyway.

**Comment by bgrant0607** [comment]
One minor comment. Otherwise LGTM. Thanks!

File: examples/guestbook/README.md (hunk 1 above)
Comment (nikhiljindal) [inline examples/guestbook/README.md]: Done.

**Comment by bgrant0607** [comment]
LGTM

**Pull request merged**